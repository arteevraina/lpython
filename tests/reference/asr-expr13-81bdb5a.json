{
    "basename": "asr-expr13-81bdb5a",
    "cmd": "lpython --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/expr13.py",
    "infile_hash": "fae72924c71183c45d328b379dfa81aa4971b5e1a9ea668db546078f",
    "outfile": null,
    "outfile_hash": null,
    "stdout": null,
    "stdout_hash": null,
    "stderr": "asr-expr13-81bdb5a.stderr",
    "stderr_hash": "0699d234dbdd85c470cbdb001dd7447b9419e96ce3aafa7642a9edc3",
    "returncode": 2
}