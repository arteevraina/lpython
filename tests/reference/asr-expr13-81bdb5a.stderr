warning: The module 'lpython_builtin' located in $DIR/src/bin/../runtime/lpython_builtin.py cannot be loaded
 --> tests/expr13.py:9:9
  |
9 |     a = complex(3, 4) == complex(3., 4.)
  |         ^^^^^^^^^^^^^ imported here

semantic error: The symbol '_lpython_floordiv' not found in the module 'lpython_builtin'
   --> $DIR/src/bin/../runtime/lpython_builtin.py:209:15
    |
209 |     if (n_ - (n_ // 2)*2) == 0:
    |               ^^^^^^^ 
